cs/src/attributes/Attributes.cs

```diff
@@ -199,4 +199,18 @@ public SerializerAttribute(Type type)
 
         internal Type Type { get; private set; }
     }
+
+    /// <summary>
+    /// Applied to 2-pass protocol writers to indicate the implementation of <see cref="IProtocolWriter" /> used to generate the first-pass serializer
```

Comment:
I know I asked for `see` to be added in comments--and thank you for doing so--but in this comment in particular we can't use `see`, as Bond.Attributes.dll doesn't refer to Bond.dll, and we don't want it to. This particular `see` should be removed and just left as "IProtocolWriter".